src/ol/interaction/Draw.js

```diff
@@ -523,7 +523,11 @@ class Draw extends PointerInteraction {
     } else if (move) {
       pass = event.type === MapBrowserEventType.POINTERMOVE;
       if (pass && this.freehand_) {
-        pass = this.handlePointerMove_(event);
+        this.handlePointerMove_(event);
+        if (this.shouldHandle_) {
```

Comment:
Are there cases where `this.shouldHandle_` is true and `this.handlePointerMove_` wasn’t returning `false`?

Reply:
`this.handlePointerMove` always returned true, so I removed the return.